app/models/enterprise.rb

```diff
@@ -71,36 +69,28 @@ class Enterprise < ActiveRecord::Base
   validate :name_is_unique
   validates :sells, presence: true, inclusion: {in: SELLS}
   validates :address, presence: true, associated: true
-  validates :email, presence: true
   validates_presence_of :owner
   validates :permalink, uniqueness: true, presence: true
   validate :shopfront_taxons
   validate :enforce_ownership_limit, if: lambda { owner_id_changed? && !owner_id.nil? }
   validates_length_of :description, :maximum => 255
 
-
-  before_save :confirmation_check, if: lambda { email_changed? }
-
   before_validation :initialize_permalink, if: lambda { permalink.nil? }
   before_validation :ensure_owner_is_manager, if: lambda { owner_id_changed? && !owner_id.nil? }
-  before_validation :ensure_email_set
   before_validation :set_unused_address_fields
   after_validation :geocode_address
 
   after_touch :touch_distributors
+  before_create :set_default_contact
   after_create :relate_to_owners_enterprises
-  # TODO: Later versions of devise have a dedicated after_confirmation callback, so use that
-  after_update :welcome_after_confirm, if: lambda { confirmation_token_changed? && confirmation_token.nil? }
-  after_create :send_welcome_email, if: lambda { email_is_known? }
+  after_create :send_welcome_email
 
   after_rollback :restore_permalink
 
 
   scope :by_name, order('name')
   scope :visible, where(visible: true)
-  scope :confirmed, where('confirmed_at IS NOT NULL')
-  scope :unconfirmed, where('confirmed_at IS NULL')
-  scope :activated, where("confirmed_at IS NOT NULL AND sells != 'unspecified'")
+  scope :activated, where("sells != 'unspecified'")
```

Comment:
Okay, that's much more clear now. I'll take a look at the `receives_notifications` idea. 

On a separate note, I've changed my mind about making `owner` just an `enterprise_role` flag, it has lots more usages throughout the codebase and would be much more tricky to untangle.